.github/workflows/pytest.yml

```diff
@@ -26,10 +26,11 @@ jobs:
         os: [macOS-latest, ubuntu-latest, windows-latest]
         python-version: ["3.9", "3.10", "3.11", "3.12"]
         chemlib: [obabel, rdkit]
-        graphlib: [nx, gt]
+        graphlib: [nx, gt, all]
         exclude:
         # graph-tools does not work on Windows
         - {os: "windows-latest", graphlib: "gt"}
+        - {graphlib: "all", chemlib: "openbabel"}
```

Comment:
```suggestion
        - {graphlib: "all", chemlib: "obabel"}
```

Reply:
This is the reason for the latest CI failure, I think. (The configuration is not skipped.)

Reply:
> This is the reason for the latest CI failure, I think. (The configuration is not skipped.)

Yeah, I noticed it wasn't getting skipped. Dumb typo on my part!